packages/lwc-language-server/scripts/convert.js

```diff
@@ -50,26 +50,71 @@ const tags = Object.keys(data).map(key => {
     };
 });
 
+//make globalAttribute changes here, not in standard-lwc.json as they'll be overwritten
 const globalAttributes = [
     {
         name: 'for:each',
-        decription: 'Renders the element or template block multiple times based on the expression value.',
+        description: 'Renders the element or template block multiple times based on the expression value.',
+        references: [
+            {
+                name: 'Salesforce',
+                url: 'https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.reference_directives',
+            },
+        ],
     },
     {
         name: 'for:item',
-        decription: 'Bind the current iteration item to an identifier.',
+        description: 'Bind the current iteration item to an identifier.',
+        references: [
+            {
+                name: 'Salesforce',
+                url: 'https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.reference_directives',
+            },
+        ],
     },
     {
         name: 'for:index',
-        decription: 'Bind the current iteration index to an identifier.',
+        description: 'Bind the current iteration index to an identifier.',
+        references: [
+            {
+                name: 'Salesforce',
+                url: 'https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.reference_directives',
+            },
+        ],
     },
     {
         name: 'if:true',
-        decription: 'Renders the element or template if the expression value is thruthy.',
+        description: 'Renders the element or template if the expression value is truthy.',
+        references: [
+            {
+                name: 'Salesforce',
+                url: 'https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.reference_directives',
+            },
+        ],
     },
     {
         name: 'if:false',
-        decription: 'Renders the element or template if the expression value is falsy.',
+        description: 'Renders the element or template if the expression value is falsy.',
+        references: [
+            {
+                name: 'Salesforce',
+                url: 'https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.reference_directives',
+            },
+        ],
+    },
+    {
+        name: 'iterator:it',
+        description: {
+            kind: 'markdown',
+            value:
+                'Bind the current iteration item to an identifier. Contains properties (`value`, `index`, `first`, `last`) that let you apply special behaviors to certain items.',
+        },
+        references: [
+            {
+                name: 'Salesforce',
+                url: 'https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.reference_directives',
+            },
+        ],
```

Comment:
dumb q: are these dups for what's in [standard-lwc.json](https://github.com/forcedotcom/lightning-language-server/pull/488/files#diff-e30348f58226e23daf10a57ff87c0b636112f69a5de89aa0434c6344403d961d). If so what don't we just import that here and use it?

Reply:
So we have a script that runs every time we build that calls `convert.js`, and that is what then populates `standard-lwc.json`. I was trying to dig up if there was still a purpose to having this separately, but wasn't able to find a clear answer in any conversation/PR/commit message. That said, globalAttributes goes back a long time in this file, and I do know for sure that this overwrites whatever we put in to standard-lwc.

Reply:
gotcha